index.js

```diff
@@ -22,6 +23,25 @@ function validatePaymentDataMiddleWare(req, res, next) {
     next();
 };
 
+function getAccountBalanceData(inPayments) {
+    const inPaymentsAmount = inPayments.map(payment => payment.amount);
```

Comment:
nope, this is rubbish to be honest :) 
you should do it similar way as you calculate total amount in PaymentsTable at the Front-End side.
simply reuse the functions from Front-End ;)